NEWS.md

```diff
@@ -1,3 +1,11 @@
+# checklist 0.2.7
+
+* fixes a note about `"MIT"` license
+* the Dockerimage uses the same dictionaries as the local installation
+* add a German dictionary
+* Spell check `roxygen2` tags in `.R` files.
+* Don't spell check `.Rd` files generated by `roxygen2`.
+
```

Comment:
I suggest to also mention some improvements that much improve the user-friendlyness.

- the updates to `setup_project`, `check_project`, ... which now use interactive questions to guide the user through the process
- the simplified instructions in `getting started` vignette, because a personal access token is no longer needed.

Maybe even consider bumping version to 0.3.0?

Reply:
NEWS is updated